src/netbase.cpp

```diff
@@ -132,12 +132,12 @@ std::vector<std::string> GetNetworkNames(bool append_unroutable)
     return names;
 }
 
-static bool LookupIntern(const std::string& name, std::vector<CNetAddr>& vIP, unsigned int nMaxSolutions, bool fAllowLookup, DNSLookupFn dns_lookup_function)
+static std::vector<CNetAddr> LookupIntern(const std::string& name, unsigned int nMaxSolutions, bool fAllowLookup, DNSLookupFn dns_lookup_function)
 {
-    vIP.clear();
+    std::vector<CNetAddr> vIP;
```

Comment:
We don't really use this naming style anymore for new code, perhaps just `addresses`?

```suggestion
    std::vector<CNetAddr> addresses;
```

Reply:
Cool, gonna adopt it